src/lib/integration_test_cloud_engine/graphql_polling_log_engine.ml

```diff
@@ -33,13 +33,9 @@ let event_reader { event_reader; _ } = event_reader
 
 let parse_event_from_log_entry ~logger log_entry =
   let open Or_error.Let_syntax in
-  let open Json_parsing in
   Or_error.try_with_join (fun () ->
-      let payload = Yojson.Safe.from_string log_entry in
       let%map event =
-        let%bind msg =
-          parse (parser_from_of_yojson Logger.Message.of_yojson) payload
-        in
+        let%bind msg = Util.log_entry_from_string log_entry in
```

Comment:
you can make this into a helper function if you like, but don't put it in util.ml. put the helper function in this same file, graphql_polling_log_engine.ml.

util.ml is more for functions which are going to be shared across files, this log parsing stuff is pretty specific to the log engines and we only use this one log engine atm

Reply:
@QuiteStochastic I'm also using this method in test_common module in test_executive app. Will it be available to that module after i move this method to graphql_polling_log_engine.ml (without adding an interface method to intf in integration_test_lib )  ?